src/Illuminate/Translation/Translator.php

```diff
@@ -160,9 +160,9 @@ public function get($key, array $replace = [], $locale = null, $fallback = true)
             // the translator was instantiated. Then, we can load the lines and return.
             $locales = $fallback ? $this->localeArray($locale) : [$locale];
 
-            foreach ($locales as $locale) {
+            foreach ($locales as $_locale) {
                 if (! is_null($line = $this->getLine(
-                    $namespace, $group, $locale, $item, $replace
+                    $namespace, $group, $_locale, $item, $replace
```

Comment:
Use `$languageLineLocale` instead here. `$_` isn't a syntax we use anywhere.

Reply:
Sure. I was wondering about it myself.

Thanks for the feedback; done in: [549de01](https://github.com/laravel/framework/pull/49900/commits/549de01ea9f7fa17b272d9baa04be0fe186b9c26)